### ISSUE TITLE
Wrong indentation at fed_ixi/benchmark line 144-154

### ISSUE THREAD

**Issue by Arctic-Xiangjian** [owkin/FLamby#249]
https://github.com/owkin/FLamby/blob/cc3ab3427a13f71283a5540720e7915be618190b/flamby/datasets/fed_ixi/benchmark.py#L144

Hello, I notice that in the above position, you make the result become np.array() which will make next different random seed experiment raise an error. that 'np.array() have no append'.

If delete the indentation of line 144-154. It will give the right output.

Here is the fix code.
```      
        results.append(current_results_dict["client_test_0"])
    results = np.array(results)
    if log:
        for i in range(results.shape[0]):
            writer = SummaryWriter(log_dir=f"./runs/tests_seed{SEEDS[i]}")
            writer.add_scalar("DICE coefficient", results[i], 0)

    print("Benchmark Results on pooled IXI:")
    print(
        f"mDICE on {len(SEEDS)} runs: {results.mean():.2%} \\pm {results.std():.2%}"
    )
```

**Comment by jeandut** [comment]
Thank you very much @Arctic-Xiangjian, we'll change that ASAP.

**Comment by jeandut** [comment]
See #254

**Comment by jeandut** [comment]
shoud be fixed in #254
